wrapper/src/main/java/com/splunk/support/lambda/configuration/SplunkConfiguration.java

```diff
@@ -90,15 +73,26 @@ private static Level getOtelLibLogLevel() {
     return Level.WARNING;
   }
 
-  private static void setDefaultValues() {
-    setDefaultValue("otel.propagators", "b3");
-    setDefaultValue("otel.exporter", "jaeger-thrift");
-    setDefaultValue("otel.exporter.jaeger.endpoint", "http://localhost:9080/v1/trace");
+  private static void setDefaults() {
+    // by default no metrics are exported
+    setDefaultValue("otel.metrics.exporter", "none");
+
+    // jaeger-thrift defaults
+    setDefaultValue("otel.traces.exporter", "jaeger-thrift-splunk");
+    setDefaultValue(OTEL_EXPORTER_JAEGER_ENDPOINT, "http://localhost:9080/v1/trace");
     setDefaultValue("otel.exporter.jaeger.service.name", "OtelInstrumentedLambda");
```

Comment:
`otel.exporter.jaeger.service.name` is no more - do we even need to set a default service name here? Otel SDK has some non-empty default

Reply:
![image](https://user-images.githubusercontent.com/68234081/109157353-2ffcb600-7772-11eb-9792-0884b9cb2b97.png)